.circleci/config.yml

```diff
@@ -19,9 +19,9 @@ jobs:
 
   test_integration:
     machine:
-      image: circleci/classic:201808-01
+      image: ubuntu-2004:202201-02
     environment:
-      K8S_VERSION: v1.17.17
+      K8S_VERSION: v1.18.10
```

Comment:
didn't know a version is coded here. thought it'd be good to match prod/stg.

Reply:
Hmm, tried rebuilding and failed again with:

```
#!/bin/bash -eo pipefail
sudo -E minikube start --vm-driver=none --cpus 2 --memory 2048 --kubernetes-version=${K8S_VERSION} &> $HOME/minikube.log 2>&1 < /dev/null


Exited with code exit status 86
CircleCI received exit code 86
```

Looking up the [error code](https://minikube.sigs.k8s.io/docs/contrib/errorcodes/#error-codes-specific-to-the-guest-host): `86: ExGuestUnsupported`
Unclear to me what the issue is. Perhaps minikube version needs a bump too?

Reply:
interesting! this [doc](https://minikube.sigs.k8s.io/docs/contrib/errorcodes/) says about the error code:
```
“GUEST_MISSING_CONNTRACK” (Exit code ExGuestUnsupported)
minikube could not find conntrack on the host, which is required from Kubernetes 1.18 onwards
```

Reply:
Nice.

Reply:
[resolved](https://app.circleci.com/pipelines/github/artsy/hokusai/543/workflows/a61db686-357f-4b12-8c70-d2f6cd51737c)!